node/src/chain_spec/flashbox.rs

```diff
@@ -217,7 +217,7 @@ fn testnet_genesis(
 
     let para_ids: Vec<_> = para_ids
         .into_iter()
-        .map(|(para_id, genesis_data, _boot_nodes)| (para_id, genesis_data))
+        .map(|(para_id, genesis_data, _boot_nodes)| (para_id, genesis_data, None))
```

Comment:
Maybe we should find a way to pass them as parathreads. Maybe the cleanest is with a new command line argument dont you think? Or alternatively, something in the spec that indicates whether it is a parathread..